quinn-proto/src/frame.rs

```diff
@@ -871,9 +891,39 @@ impl Datagram {
     }
 }
 
+#[derive(Debug, Copy, Clone)]
+pub(crate) struct AckFrequency {
+    pub(crate) sequence: u64,
+    pub(crate) ack_eliciting_threshold: u64,
+    pub(crate) request_max_ack_delay: u64,
+    pub(crate) reordering_threshold: u64,
```

Comment:
Let's use `VarInt` rather than `u64` to reduce opportunity for error elsewhere. We're not totally consistent about that, but stronger types are generally preferred, making e.g. the `u64::MAX` bug above more obvious. This also allows `.write(x)` and `.get()` to be used directly.